src/test/java/com/epam/tc/hw2/ex1/FirstExercise.java

```diff
@@ -0,0 +1,113 @@
+package com.epam.tc.hw2.ex1;
+
+import com.epam.tc.hw2.BaseClass;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class FirstExercise extends BaseClass {
+
+    SoftAssertions softAssert = new SoftAssertions();
```

Comment:
There is no `softAssert.assertAll();` in test method. Without assertAll soft asserts won't perform checks

And please move softAssertions initialization to the test method

Reply:
Added assertAll